contracts/mocks/AccessManagedTarget.sol

```diff
@@ -15,4 +16,13 @@ abstract contract AccessManagedTarget is AccessManaged {
     function fnUnrestricted() public {
         emit CalledUnrestricted(msg.sender);
     }
+
+    function setIsConsumingScheduledOp(bool isConsuming) external {
+        // Memory layout is 0x....<_consumingSchedule (boolean)><authority (address)>
+        _consumingSchedule().value |= bytes32(uint256(isConsuming ? 1 : 0)) << 161;
```

Comment:
if bool `isConsuming` is false, and if the bit is already set, this function will not reset it.
```suggestion
        bytes32 mask = 1 << 161;
        if (isConsuming) {
            _consumingSchedule().value |= mask;
        } else {
            _consumingSchedule().value &= ~mask;
        }
```

Reply:
I checked it out and yes, you're right. I'm updating with another bug I found from this.